### ISSUE TITLE
Wrong GPU detected. RX 470 instead of RX 480.

### ISSUE THREAD

**Issue by BobbyBabes** [dylanaraps/neofetch#460]
## Description

GPU is incorrectly displayed as AMD Radeon RX 470. This should be AMD Radeon RX 480.
Looking at the attached screenshot, the AMD website displays the
RX 480 with PCI-DeviceID and PCI-RevisionID : 0x67DF 0xC7
RX 470 with PCI-DeviceID and PCI-RevisionID : 0x67DF 0xCF
"lspci | grep VGA" outputs PCI-RevisionID : 0xc7
"grep -e Chipset /var/log/Xorg.0.log" outputs PCI-DeviceID : 0x67df

P.S. I'm not too worried about this. It's not a biggie. Just reporting it.

## System Information
GPU: AMD RX 480 (POLARIS10) PCI-DeviceID: 0x67df PCI-RevisionID: 0xc7
#### Neofetch version
2.0
#### Operating system
Arch Linux x86_64
## Screenshot
![neofetch-wrong-gpu_20161126001228](https://cloud.githubusercontent.com/assets/22870070/20636860/4b015f0c-b375-11e6-9a09-aa3101c27473.png)

## Verbose log

http://pastebin.com/cwqckf9e

**Comment by dylanaraps** [comment]
Thanks for reporting this!

This seems to occur because we delete everything after the last `/` in the lspci output. I'll edit it so that we still include the bit of the gpu info.

The output will then look like this on your machine:

```sh
GPU: AMD Radeon RX 470/480
```

This is the best I can do with what `lspci` gives us unfortunately.

**Comment by dylanaraps** [comment]
Can you send me the output of this command?

```sh
PATH="/sbin:$PATH" lspci -mm | awk -F '\\"|\\" \\"' '/3D|VGA/ {print $3 " " $4}'
```

**Comment by BobbyBabes** [comment]
Output of that command is in the screenshot below.
The best you can do is good enough for me. :)

![neofetch-wrong-gpu_20161126104422](https://cloud.githubusercontent.com/assets/22870070/20639504/ab926c6c-b3c5-11e6-9d9e-58bee8eeb1fb.png)

**Comment by dylanaraps** [comment]
The issue should be fixed now. Sorry for the delay. :)

**Comment by BobbyBabes** [comment]
No worries.

The fix doesn't seem to have changed anything for me.
(Going to sleep now, because it's in the still of the night here.)

git pull
make install
See screenshot and log file.

Verbose log : http://pastebin.com/jjwL6Mig

![neofetch-gpu-fix-not-ok-2](https://cloud.githubusercontent.com/assets/22870070/20690524/70bd5dce-b5cb-11e6-9d03-84d08be4717d.png)

![neofetch-gpu-fix-not-ok](https://cloud.githubusercontent.com/assets/22870070/20690362/6ddf8736-b5ca-11e6-9f3b-0977360495e0.png)

**Comment by dylanaraps** [comment]
Neofetch caches the GPU name since `lspci` can be slow. Try deleting the cached value and let me know if it works. `rm /tmp/neofetch/gpu`

Sorry, I should've mentioned this in my previous comment. If the issue still persists then I'll take a closer look at this.

**Comment by BobbyBabes** [comment]
I checked, but I have no ``/tmp/neofetch/gpu`` file or folder.
Not before and not after running ``make install``.

**Comment by dylanaraps** [comment]
Weird, the fix works on my machine. (I hardcoded my GPU value to your `lspci` screenshot above.)

**Comment by BobbyBabes** [comment]
OK, i found out what's going wrong.
It's because of my silliness. For which I apologise.

I copied the repo from its local share ``/srv/dvcs/github/neofetch`` into /tmp with ``sudo rsync -avHEAX /srv/dvcs/github/neofetch /tmp``.
It has root as owner and group, because I ``rsync``ed with sudo. Which was silly and caused the problem.
(I normally use ``cd /tmp; git clone --depth 1 file://designbox/srv/dvcs/github/neofetch``)

I then installed from the repo folder named ``/tmp/neofetch``.
When I ran neofetch as a regular user it couldn't write to ``/tmp/neofetch``.
So its gpu file could not be created.

After deleting the folder ``/tmp/neofetch``, neofetch worked OK.

![neofetch-gpu-fix-ok](https://cloud.githubusercontent.com/assets/22870070/20704882/299cc424-b621-11e6-8b2f-35f1973b846f.png)
